spring-cloud-starter-stream-sink-jdbc/README.adoc

```diff
@@ -26,6 +26,8 @@ $$jdbc.columns$$:: $$The comma separated colon-based pairs of column names and S
  Names are used at initialization time to issue the DDL.$$ *($$String$$, default: `$$payload:payload.toString()$$`)*
 $$jdbc.initialize$$:: $$'true', 'false' or the location of a custom initialization script for the table.$$ *($$String$$, default: `$$false$$`)*
 $$jdbc.table-name$$:: $$The name of the table to write into.$$ *($$String$$, default: `$$messages$$`)*
+$$jdbc.batch-size$$:: $$Threshold in number of messages when data will be flushed to database table.$$ *($$Integer$$, default: `$$1$$`)*
+$$jdbc.idle-timeout$$:: $$Idle timeout in milliseconds when data is automatically flushed to database table.$$ *($$Long$$, default: `$$-1$$`)*
```

Comment:
I think we also need to add a paragraph with general explanation how to achieve a batch inserts during regular streaming process.

Reply:
A added a short description on how to use batch inserts, see commit 31054aed41b3955d7ec1f8c50e06038a1831a2f9 .